.github/workflows/build.yml

```diff
@@ -0,0 +1,82 @@
+name: build
+
+on:
+  push: {}
+  pull_request: {}
+  # allow rebuilding without a push
+  workflow_dispatch: {}
```

Comment:
I would suggest adding a weekly or monthly [`on: schedule:`](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#schedule) run to keep the sstate fresh and to check if everything still works with the current master of all the other layers.

But that's an unrelated change that could be done in a separate PR.